main.go

```diff
@@ -59,12 +60,14 @@ func main() {
 	var enableLeaderElection bool
 	var rulesConfigmapName string
 	var rulesConfigmapNamespace string
+	var rulesFileName string
 
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&rulesConfigmapName, "rulesConfigmapName", "oathkeeper-rules", "Name of the Configmap that stores Oathkeeper rules.")
 	flag.StringVar(&rulesConfigmapNamespace, "rulesConfigmapNamespace", "oathkeeper-k8s-controller-system", "Namespace of the Configmap that stores Oathkeeper rules.")
+	flag.StringVar(&rulesFileName, "rulesFileName", "access_rules.json", "Name of the file with converted Oathkeeper rules")
```

Comment:
The controller doesn't create any file (yet), it's just a name of the key in the map. I think that config flag should be named accordingly: `mapKey` `keyName` or something like that.